pkg/sti/create/templates/docker.go

```diff
@@ -0,0 +1,42 @@
+package templates
+
+const Dockerfile = `
+# {{.ImageName}}
+FROM centos:centos7
+
+# TODO: Install required packages:
+#
+# RUN yum install -y --enablerepo=centosplus epel-release <packages>
+
+
+# Add STI usage script
+ADD ./.sti/bin/usage /usage
+
+# You can add any other required files/configurations here
+# ADD conf /app-root/conf
+
+# TODO: Define the URL from where the default STI script will be fetched:
+#
+# ENV STI_SCRIPTS_URL https://raw.githubusercontent.com/<repository>/master/.sti/bin
```

Comment:
I'd suggest including here both of the vars we expect from images, meaning `STI_SCRIPTS_URL` and `STI_LOCATION`.